apps/user-profile/src/app/pages/consents-page/consents-preview/consents-preview.component.html

```diff
@@ -0,0 +1,47 @@
+<h1 class="page-title mt-2">
+  {{ 'CONSENTS.TITLE' | customTranslate | translate }}
+</h1>
+<div class="user-theme">
+  <div *ngIf="!loading && unsignedConsents.length !== 0">
+    <div class="page-subtitle">
+      {{ 'CONSENTS.UNSIGNED_CONSENTS' | customTranslate | translate }}
+    </div>
+
+    <perun-web-apps-immediate-filter
+      [placeholder]="'CONSENTS.FILTER' | customTranslate | translate"
+      (filter)="applyFilterUnsigned($event)">
+    </perun-web-apps-immediate-filter>
+
+    <perun-web-apps-consents-list
+      [consents]="unsignedConsents"
+      [filterValue]="filterValueUnsigned"
+      [tableId]="'TABLE_USER_CONSENTS'"
+      (grantConsent)="grantConsent($event)"
+      (rejectConsent)="rejectConsent($event)"
+      [displayedColumns]="['status', 'name']"></perun-web-apps-consents-list>
+
+    <!--    <button mat-flat-button color="accent" class="mt-3" (click)="grantAll()">-->
+    <!--      {{ 'CONSENTS.GRANT_ALL' | customTranslate | translate }}-->
+    <!--    </button>-->
+    <div class="page-subtitle mt-4">
+      {{ 'CONSENTS.PROCESSED_CONSENTS' | customTranslate | translate }}
+    </div>
```

Comment:
This title should be outside of the `<div>` containing unsigned consents. Also probably wrap the proccesed consents in their own div and put the `*ngIf=!loading` on the div. So we don't handle each component separatly.